src/matrix/common.js

```diff
@@ -22,16 +22,16 @@ export function makeTxnId() {
 }
 
 export function isTxnId(txnId) {
-	return txnId.startsWith("t") && txnId.length === 15;
```

Comment:
Please avoid making unrelated whitespace changes. It makes it harder to see what is being changed.